delfin/drivers/dell_emc/scaleio/rest_handler.py

```diff
@@ -0,0 +1,354 @@
+                    }
+                    alert_list.append(alert_model)
+                return alert_list
+        except Exception as e:
+            LOG.error("Get Storage alerts error: %s", six.text_type(e))
+            raise e
+
+    def list_storage_host_initiators(self, storage_id):
+        initiators_list = []
+        try:
+            storage_initiators = self.get_rest_info(
+                consts.REST_SCALIO_INITIIATORS)
+            if storage_initiators:
+                for initiators_json in storage_initiators:
+                    status = initiators_json.get('mdmConnectionState')
+                    initiators_id = initiators_json['id']
+                    if 'Connected' == status:
+                        status = constants.HostStatus.NORMAL
+                    elif 'Disconnected' == status:
+                        status = constants.HostStatus.OFFLINE
+                    initiators_dict = {
+                        "name": initiators_json.get('name'),
+                        "storage_id": storage_id,
+                        "native_storage_host_initiator_id": initiators_id,
+                        "wwn": initiators_id,
+                        "status": status,
+                        "native_storage_host_id": initiators_json.get(
+                            'protectionDomainId'),
+                    }
+                    initiators_list.append(initiators_dict)
+                return initiators_list
+        except Exception as e:
+            LOG.error("Get Storage initiators error: %s", six.text_type(e))
+            raise e
+
+    def list_storage_hosts(self, storage_id):
+        host_list = []
+        try:
+            storage_hosts = self.get_rest_info(consts.REST_SCALIO_HOSTS)
+            if storage_hosts:
+                for host_json in storage_hosts:
+                    status = host_json.get('mdmConnectionState')
+                    if 'Connected' == status:
+                        status = constants.HostStatus.NORMAL
+                    elif 'Disconnected' == status:
+                        status = constants.HostStatus.OFFLINE
+                    ip_list = host_json.get('ipList')
+                    ip_address = ''
+                    for ip in ip_list:
+                        ip_address = ip.get('ip')
+                    host_dict = {
+                        "name": host_json.get('name'),
+                        "description": host_json.get('sdcGuid'),
+                        "storage_id": storage_id,
+                        "native_storage_host_id":
+                            host_json.get('id'),
+                        "os_type": host_json.get('osType'),
+                        "status": status,
+                        "ip_address": ip_address
+                    }
+                    host_list.append(host_dict)
+                return host_list
+        except Exception as e:
+            LOG.error("Get Storage hosts error: %s", six.text_type(e))
+            raise e
```

Comment:
Exceptions cannot be thrown directly

Reply:
Throw the corresponding exception
![image](https://user-images.githubusercontent.com/88076879/170652427-fffded97-dae0-4cdb-bf5c-2e9226894508.png)